### PR TITLE
test: isolate recs list tests

### PR DESCRIPTION
**Pull request by mrrajamanickam-coveo**
In an effort to stabilize recs list tests and prep for cy v12 - replace all `before` with `beforeEach`.

Other changes: 
* Quantic: Minor readme typo fixes and pkg json cleanup FYI @erocheleau 

https://coveord.atlassian.net/browse/KIT-2440

### CODE REVIEW

**Review by olamothe — APPROVED** [review]

**Comment by mrrajamanickam-coveo** [comment]
@erocheleau @mmitiche Can you please review this PR. It has some minor quantic changes.

File: packages/quantic/README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@
 
 ### Install VSCode Extension
 
-[vscode Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode)
+Optionally install the [VSCode Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) if you do not want use the equivalent `sfdx-cli` commands to setup.
```
Comment (mrrajamanickam-coveo) [inline packages/quantic/README.md:27]: Unless there are quantic dev workflows other than setup that use VSCode without (specifying) alternate sfdx commands.

File: packages/quantic/package.json [hunk 2]
```diff
@@ -24,9 +24,9 @@
     "test:unit:watch": "lwc-jest --watch",
     "test:unit:coverage": "lwc-jest --coverage",
     "validate:types": "tsc",
-    "e2e:watch": "npx cypress open",
-    "e2e": "npx cypress run --browser chrome",
-    "e2e:detailed": "npx cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
+    "e2e:watch": "cypress open --browser chrome --e2e",
```
Comment (mmitiche) [inline packages/quantic/package.json:27]: I personally liked that we didn't specify chrome directly in the command in order to get the ability to easily switch the browser where the tests will be executed directly from the UI when I'm watching the tests locally:

e2e:watch

**Review by mmitiche — APPROVED** [review]
LGTM!
Just a small preference I mentioned.

File: packages/quantic/package.json (hunk 2 above)
Comment (mrrajamanickam-coveo) [reply packages/quantic/package.json:27]: It is still possible to switch to a different browser - this just launches cypress directly into choosing the test suite dialog - its just one click away to get back to choose a browser dialog.

**Pull request merged**